src/script/script_gui.cpp

```diff
@@ -896,9 +830,41 @@ struct ScriptDebugWindow : public Window {
 
 	void DrawWidget(const Rect &r, int widget) const override
 	{
-		if (script_debug_company == INVALID_COMPANY) return;
+		switch (widget) {
+			case WID_SCRD_LOG_PANEL:
+				this->DrawWidgetLog(r);
+				break;
 
-		if (widget != WID_SCRD_LOG_PANEL) return;
+			default:
+				if (IsInsideBS(widget, WID_SCRD_COMPANY_BUTTON_START, MAX_COMPANIES)) {
+					this->DrawWidgetCompany(r, widget, WID_SCRD_COMPANY_BUTTON_START);
+				}
+				break;
+		}
+	}
+
+	/**
+	 * Draw a company button icon.
+	 * @param r Rect area to draw within.
+	 * @param widget Widget index to start.
+	 * @param start Widget index of first company button.
+	 */
+	void DrawWidgetCompany(const Rect &r, int widget, int start) const
```

Comment:
```suggestion
	void DrawWidgetCompanyButton(const Rect &r, int widget, int start) const
```

Maybe? To make it clear it's not the log of that AI, and just the button.